nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/admin/HealthCheckConfigurer.java

```diff
@@ -144,8 +147,30 @@ public Object run(final HealthCheckServiceConfiguration healthCheckServiceConfig
                 configureDynamically();
             }
         }
+
+        enableLogNotifier(actionReport, context);
     }
 
+
+    private void enableLogNotifier(ActionReport actionReport, AdminCommandContext context) {
+        CommandRunner runner = serviceLocator.getService(CommandRunner.class);
+        ActionReport subReport = context.getActionReport().addSubActionsReport();
+
+        CommandRunner.CommandInvocation inv = runner.getCommandInvocation("healthcheck-log-notifier-configure", subReport, context.getSubject());
+
+        ParameterMap params = new ParameterMap();
+        params.add("dynamic", dynamic.toString());
+        params.add("target", target);
+        params.add("enabled", notifierEnabled.toString());
```

Comment:
In line with my comment about removing the default value from this parameter, this will need a null check:

```
if (notifierEnabled != null) {
    params.add("enabled", notifierEnabled.toString());
}
```